Week-1/Session-02-HTML_Deployment/Submission/Intermediate/Exercise-01/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hometown Webpage</title>
+  </head>
+  <body>
+    <h1>sometown, USA</h1>
```

Comment:
not containing section tags as described in point 3

Reply:
updated as per suggestion.